.github/workflows/build_push.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Push Docker Images
```

Comment:
so everytime we add a new dockerfile, we need to add a new docker push github workflow?
Can you add this to the `developing cookbook` documentation

Reply:
i don't think we have that right?  we only have it for flyte https://github.com/flyteorg/flyte/blob/master/rsts/contributor/docs/index.rst